ecl/ecl-package/CMakeLists.txt

```diff
@@ -56,7 +56,12 @@ include_directories (
 ADD_DEFINITIONS( -D_CONSOLE )
 
 HPCC_ADD_EXECUTABLE ( ecl-packagemap ${SRCS} )
-add_dependencies ( ecl-packagemap espscm ws_packageprocess )
+
+set( WS_PACKAGEPROCESS_DEP )
+if( NOT CLIENTTOOLS_ONLY )
```

Comment:
That would be a workaround but the correct fix is to state the dependency that actually exists - i.e. a dependency on the scm file

Is there a reason the relevant scm files are not included in the espscm target?